Predictors/3d_kPCA.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+# This import is used even though PyCharm thinks it is not
+# In case deleted it is this import "from mpl_toolkits.mplot3d import Axes3D"
+from mpl_toolkits.mplot3d import Axes3D
+from numpy import array
+from sklearn.decomposition import KernelPCA
+from sklearn.feature_extraction.text import CountVectorizer
+
+from data.import_data import import_data
+
+train, ts = import_data()
+
+# Storing the entire training text in a list
+text = list(train.text.values)
+authors = array(list(train.author.values))
+
+tf_vectorizer = CountVectorizer(max_df=0.95,
+                                min_df=2,
+                                # stop_words='english',
+                                decode_error='ignore')
+
+tf = tf_vectorizer.fit_transform(text)
+
+# print(tf)
+
+rbf_pca = KernelPCA(n_components=3, kernel="sigmoid", gamma=0.04)
```

Comment:
A lot of this code is duplicated between this file and kPCA, maybe pull into a separate function?

Reply:
Although some code is duplicated my thought was as they work in different dimensions it may be better to keep them separate as it's likely the code will need changing